src/phoenix/server/api/types/EvaluationSummary.py

```diff
@@ -1,13 +1,15 @@
 import math
 from functools import cached_property
-from typing import List, Optional, Tuple, cast
+from typing import List, Optional, Sequence, Tuple, Union, cast
 
 import pandas as pd
 import strawberry
 from pandas.api.types import CategoricalDtype
 from strawberry import Private
 
-import phoenix.trace.v1 as pb
+from phoenix.db import models
+
+annotation_type = Union[models.SpanAnnotation, models.TraceAnnotation]
```

Comment:
It's more conventional for type names to be upper camel case, so `AnnotationType` (although `EvaluationType`) might make more sense here.